### PR TITLE
use allowed mentions to prevent mentioning everyone

### PR DESCRIPTION
**Pull request by onerandomusername**
closes GH-39

### CODE REVIEW

File: bot/bot.py [hunk 1]
```diff
@@ -22,8 +22,25 @@ def __init__(self) -> None:
...
+            ]
+        ]
+
+        super().__init__(
+            command_prefix=constants.PREFIX,
+            intents=intents,
+            allowed_mentions=AllowedMentions(
+                everyone=None,
+                users=True,
+                roles=roles,
```
Comment (gustavwilliam) [inline bot/bot.py:40]:
```suggestion
        allowed_roles = [
            Object(r)
            for r in [
                constants.Roles.steering_council,
                constants.Roles.moderators,
                constants.Roles.gurkult_lords,
            ]
        ]

        super().__init__(
            command_prefix=constants.PREFIX,
            intents=intents,
            allowed_mentions=AllowedMentions(
                everyone=None,
                users=True,
                roles=allowed_roles,
```
Let's make it explicit.

**Review by gustavwilliam — CHANGES_REQUESTED** [review]
Pure code review. Just one comment, but otherwise LGTM!

**Review by gustavwilliam — APPROVED** [review]
LGTM, code review-wise.

File: bot/bot.py (hunk 1 above)
Comment (onerandomusername) [reply bot/bot.py]: https://github.com/gurkult/gurkbot/pull/158/commits/13db234f55c490ec54c38a658c1dd06f9dbec158

**Review by Akarys42 — APPROVED** [review]

**Pull request merged**